server/index.js

```diff
@@ -19,11 +24,32 @@ app.get('/houses/*', (req, res) => {
           res.send(rows); 
         })
         .then((res) => {
-          console.log(res);
+          //console.log(res);
+          conn.end();
+        })
+        .catch(err => {
+          //console.log(`Error: ${err}`);
```

Comment:
Errors should be sent back to the client with the appropriate status code and error message.

Reply:
Additionally, you should at least log any 5xx errors to `stderr` with `console.error`. This will allow a logging tool to capture debug information in the future.